I/ITKIOWrapper/build_tarballs.jl

```diff
@@ -0,0 +1,59 @@
+# Note: this script will require BinaryBuilder.jl v0.3.0 or greater
+using BinaryBuilder, Pkg
+julia_versions = [v"1.7", v"1.8", v"1.9", v"1.10", v"1.11"]
+name = "ITKIOWrapper"
+version = v"1.0.0"  # Update this to your package version
+
+# The following lines should be updated to reflect the actual URL and revision of your ITKWrapper package
+
+# Collection of sources required to build ITKWrapper
+sources = [
+    DirectorySource("./src"),
+]
+uuid = Base.UUID("a83860b7-747b-57cf-bf1f-3e79990d037f")
+delete!(Pkg.Types.get_last_stdlibs(v"1.6.3"), uuid)
+
+# Bash recipe for building across all platforms
+script = raw"""
+export CXXFLAGS="-I${includedir}/julia $CXXFLAGS"
+export CFLAGS="-I${includedir}/julia $CFLAGS"
+mkdir -p build/
+cmake -B build -S . \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DBUILD_SHARED_LIBS:BOOL=ON
+cmake --build build --parallel ${nproc}
+cmake --install build
+install_license /usr/share/licenses/MIT
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+include("../../L/libjulia/common.jl")
+platforms = vcat(libjulia_platforms.(julia_versions)...)
+platforms = expand_cxxstring_abis(platforms)
+
+platforms = supported_platforms()
+
+#Filtering all the platforms, that ITK filters https://github.com/JuliaPackaging/Yggdrasil/blob/master/I/ITK/build_tarballs.jl#L42-L53
+filter!(p -> !(arch(p) == "i686"), platforms)
+filter!(!Sys.isapple, platforms)
+filter!(!Sys.isfreebsd, platforms)
+filter!(p -> !(arch(p) == "x86_64" && libc(p) == "musl"), platforms)
+
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libITKIOWrapper", :libITKIOWrapper),
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency("ITK_jll"),
+    Dependency("libcxxwrap_julia_jll"),
```

Comment:
I would strongly recommend to put a `compat` restriction on this. Just grep for other build recipes in Yggdrasil that use `libcxxwrap_julia_jll` to see how to do that. E.g. `J/jlqml/build_tarballs.jl`

Reply:
Thanks for reminding for the compat restriction argument. You review is greatly beneficial, thanks :)